spacemacs/packages.el

```diff
@@ -136,6 +137,35 @@ which require an initialization must be listed explicitly in the list.")
       (setq ace-jump-mode-scope 'global)
       (evil-leader/set-key "`" 'ace-jump-mode-pop-mark))))
 
+(defun spacemacs/init-ace-link ()
+  (use-package ace-link
+    :defer t
+    :init (ace-link-setup-default)
```

Comment:
do you think it's a good idea to list all `ace-link` interactive functions in spacemacs? this implies if `ace-link` adds more functions, `spacemacs` has to be updated and includes all those functions as well.